public/src/actions/signupActions.js

```diff
@@ -1,24 +0,0 @@
-  /* eslint-disable */
-import * as types from './actionTypes';
-import store from '../store/store';
-import { hashHistory } from 'react-router';
-const register = (data)=>{
-  fetch('/signup',{
-    method: 'POST',
-    body:JSON.stringify(data),
-    credentials: 'include'
-  })
-  .then((response)=>{
-    return  response.json()
-  })
-.then((response)=>{
-    if(response.statusCode === 200 && response.msg === 'User regestered'){
```

Comment:
A bit risky to use response.msg string as part of the if statement, because if there is a typo or a change in the string, this callback will not be executed. Consider instead either storing the string as a variable that is imported both in the handler and this file, or using a different way of checking the response.